packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -145,4 +146,23 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
 
     return WEB_RTC_PREFIX + this.agentConfig.agentId;
   }
+
+  /**
+   * This is used for setting agent state.
+   * @param options
+   * @returns Promise<SetStateResponse>
+   * @throws Error
+   */
+
+  public async setAgentStatus(data: StateChange): Promise<SetStateResponse> {
```

Comment:
be consistant with naming. either state or status.

Reply:
Changed to `setAgentState`.